tests/test_install_requires.py

```
@@ -38,47 +38,48 @@
-	def tearDown(self):
+	def tearDown(self) -> None:
 		"""Clean up the temporary requirements file"""
 		try:
 			if os.path.exists(self.requirements_file):
 				os.remove(self.requirements_file)
 		finally:
-			super(TestParseRequirements, self).tearDown()
+			super(ParseRequirementsTestSuite, self).tearDown()
 
-	def write_requirements(self, content):
+	def write_requirements(self, content) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `content`

(ANN001)